ci/scripts/run-build-ic.sh

```diff
@@ -18,11 +18,9 @@ done
 # run build with release on protected branches or if a pull_request is targeting an rc branch
 if [ "${IS_PROTECTED_BRANCH:-}" == "true" ] || [[ "${CI_MERGE_REQUEST_TARGET_BRANCH_NAME:-}" == "rc--"* ]]; then
     ci/container/build-ic.sh -i -c -b
-# if an override was requested to run all bazel targets with no release
-elif [[ "${CI_PULL_REQUEST_TITLE:-}" == *"[RUN_ALL_BAZEL_TARGETS]"* ]]; then
-    ci/container/build-ic.sh -i -c -b --no-release
```

Comment:
This is not used anymore, right?

Reply:
If you open the file you see the line `ci/container/build-ic.sh -i -c -b --no-release` is still run in the `else` statement. We just don't need this explicit elif block anymore.